cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/License.hs

```diff
@@ -40,128 +41,33 @@ fromCabalLicense AllRightsReserved                         = Known "lib.licenses
-            SPDX.LGPL_2_0_or_later -> Known "lib.licenses.lgpl2Plus"
-            SPDX.LGPL_2_1_only -> Known "lib.licenses.lgpl21Only"
-            SPDX.LGPL_2_1_or_later -> Known "lib.licenses.lgpl21Plus"
-            SPDX.LGPL_3_0_only -> Known "lib.licenses.lgpl3Only"
-            SPDX.LGPL_3_0_or_later -> Known "lib.licenses.lgpl3Plus"
-            SPDX.Libpng -> Known "lib.licenses.libpng"
-            SPDX.Libtiff -> Known "lib.licenses.libtiff"
-            SPDX.LPPL_1_2 -> Known "lib.licenses.lppl12"
-            SPDX.LPPL_1_3c -> Known "lib.licenses.lppl13c"
-            SPDX.LPL_1_02 -> Known "lib.licenses.lpl-102"
-            SPDX.MIT -> Known "lib.licenses.mit"
-            SPDX.MPL_1_0 -> Known "lib.licenses.mpl10"
-            SPDX.MPL_1_1 -> Known "lib.licenses.mpl11"
-            SPDX.MPL_2_0 -> Known "lib.licenses.mpl20"
-            SPDX.MS_PL -> Known "lib.licenses.mspl"
-            SPDX.NCSA -> Known "lib.licenses.ncsa"
-            SPDX.NPOSL_3_0 -> Known "lib.licenses.nposl3"
-            SPDX.OFL_1_1 -> Known "lib.licenses.ofl"
-            SPDX.OLDAP_2_8 -> Known "lib.licenses.openldap"
-            SPDX.OpenSSL -> Known "lib.licenses.openssl"
-            SPDX.OSL_2_1 -> Known "lib.licenses.osl21"
-            SPDX.OSL_3_0 -> Known "lib.licenses.osl3"
-            SPDX.PHP_3_01 -> Known "lib.licenses.php201"
-            SPDX.PostgreSQL -> Known "lib.licenses.postgresql"
-            SPDX.Python_2_0 -> Known "lib.licenses.psfl"
-            SPDX.QPL_1_0 -> Known "lib.licenses.qpl"
-            SPDX.Ruby -> Known "lib.licenses.ruby"
-            SPDX.Sendmail -> Known "lib.licenses.sendmail"
-            SPDX.SGI_B_2_0 -> Known "lib.licenses.sgi-b-0"
-            SPDX.Sleepycat -> Known "lib.licenses.sleepycat"
-            SPDX.TCL -> Known "lib.licenses.tcltx"
-            SPDX.Unlicense -> Known "lib.licenses.unlicense"
-            SPDX.Vim -> Known "lib.licenses.vim"
-            SPDX.VSL_1_0 -> Known "lib.licenses.vsl10"
-            SPDX.Watcom_1_0 -> Known "lib.licenses.watcom"
-            SPDX.W3C -> Known "lib.licenses.w3c"
-            SPDX.WTFPL -> Known "lib.licenses.wtfpl"
-            SPDX.Zlib -> Known "lib.licenses.zlib"
-            SPDX.ZPL_2_0 -> Known "lib.licenses.zpl20"
-            SPDX.ZPL_2_1 -> Known "lib.licenses.zpl21"
-            _ ->
-              -- Licence is not in Nixpkgs.
-              -- Use the SPDX expression as a free-form license string.
-              Unknown (Just $ prettyShow expr)
-        _ ->
-          -- Not handed: the '+' suffix and user-defined licences references.
-          -- Use the SPDX expression as a free-form license string.
-          Unknown (Just $ prettyShow expr)
-    _ ->
-      -- Not handled: compound expressions, not expressible in Nixpkgs.
-      -- Use the SPDX expression as a free-form license string.
-      Unknown (Just $ prettyShow expr)
+fromSPDXLicense SPDX.NONE = Known "lib.licenses.unfree"
```

Comment:
Again, we simply conclude this has no specified license, but is open source.

Reply:
Same as above, will switch this to "lib.licenses.free"